### PR TITLE
Drop units in `eigen_est`, since `internalnorm` drops units

### PR DESCRIPTION
**Pull request by YingboMa**

### CODE REVIEW

File: src/perform_step/low_order_rk_perform_step.jl [hunk 1]
```diff
@@ -783,7 +783,7 @@ end
       ϱu += (k7[i] - k6[i])^2
       ϱd += (g7[i] - g6[i])^2
     end
-    integrator.eigen_est = sqrt(ϱu/ϱd)
+    integrator.eigen_est = ustrip(sqrt(ϱu/ϱd))
```
Comment (ChrisRackauckas) [inline src/perform_step/low_order_rk_perform_step.jl]: do we actually depend on unitful already?

**Comment by devmotion** [comment]
Just noticed that https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/blob/master/src/perform_step/low_order_rk_perform_step.jl#L781 causes a type instability, as far as I can see. I guess it should be `ϱd = zero(eltype(g7))`.

File: src/perform_step/low_order_rk_perform_step.jl [hunk 2]
```diff
@@ -778,12 +778,12 @@ end
     g6 = tmp
     g7 = u
     # Hairer II, page 22
-    ϱu, ϱd = zero(eltype(k7)), zero(eltype(k7))
+    ϱu, ϱd = zero(eltype(k7)), zero(eltype(g7))
```
Comment (devmotion) [inline src/perform_step/low_order_rk_perform_step.jl]: Sorry, my bad, I think both values should be squared as well.

File: src/perform_step/low_order_rk_perform_step.jl [hunk 3]
```diff
@@ -778,12 +778,12 @@ end
...
     g7 = u
     # Hairer II, page 22
-    ϱu, ϱd = zero(eltype(k7)), zero(eltype(k7))
+    ϱu, ϱd = zero(eltype(k7))^2, zero(eltype(g7))^2
     @inbounds for i in eachindex(k7)
       ϱu += (k7[i] - k6[i])^2
       ϱd += (g7[i] - g6[i])^2
     end
-    integrator.eigen_est = sqrt(ϱu/ϱd)
+    integrator.eigen_est = sqrt(ϱu/ϱd)*oneunit(t)
```
Comment (ChrisRackauckas) [inline src/perform_step/low_order_rk_perform_step.jl]: why would this have units then?

Comment (YingboMa) [reply src/perform_step/low_order_rk_perform_step.jl]: Multiply by `oneunit(t)` cancels units.

**Comment by YingboMa** [comment]
Bump

**Comment by ChrisRackauckas** [comment]
Rebase first.

**Pull request merged**